package/src/components/MessageInput/hooks/useAudioPreviewManager.tsx

```diff
@@ -0,0 +1,98 @@
+import { useCallback, useEffect, useState } from 'react';
```

Comment:
Shall we export the hook from the SDK for easy usage by integrators?

Reply:
Yep, that makes sense. Instead of it relying on passed `files`, can we grab the files directly in there using whichever context is responsible for them ? Or do we postprocess them as well ?

If we have to pass the files (and if we modify them ourselves) there are certain expectations that need to be documented if we're going to expose this hook.

But, I'm all for it in general 👍

Reply:
if integrators have an easy way to get these files then I'm fine; just make sure to document this properly

Reply:
Yeah its super easy through the use message composer